autopush/db.py

```diff
@@ -478,8 +515,9 @@ def __getattr__(self, name):
 
 class Message(object):
     """Create a Message table abstraction on top of a DynamoDB Table object"""
-    def __init__(self, tablename, metrics=None, boto_resource=None,
-                 max_ttl=MAX_EXPIRY):
+    def __init__(self, tablename=None, metrics=None, boto_resource=None,
+                 max_ttl=MAX_EXPIRY,
+                 table_base_string="message_"):
```

Comment:
How is the table_base_string different from the prefix?

Reply:
There's not a huge difference between the two, aside from `prefix` is used mostly in table rotation, where `table_base_string` is not.